Makefile

```diff
@@ -50,7 +50,7 @@ export RISCV_ABI
 #############################################################
 
 .PHONY: all 
-all: clean
+all: 
```

Comment:
This is a change that should definitely happen, however, I would personally leave it out for the PR:
- It's strictly not necessary for FreeRTOS functioning
- Because we haven't added all the dependency management, changing bsp/X300/platform.h wouldn't trigger a recompile so I had to manually issue a clean